presto-docs/src/main/sphinx/spill/overview.rst

```diff
@@ -44,6 +44,21 @@ memory intensive queries. It is still possible that the query runner will fail
 to divide intermediate data into chunks small enough that every chunk fits into
 memory, leading to `Out of memory` errors while loading the data from disk.
 
+Disk usage
```

Comment:
I think it is better to keep this chapter top to bottom to keep overall clarity. Section `Memory management and spill` starts with explaining the motivation for spill and describes how it works, so it is at the beginning. Then we have more practical details about memory tuning and handling issues.

I don't think that giving a recommendation on hardware configuration on the feature level can be helpful for administrators. They usually need it before installing presto and they will read the spill docs much later. Maybe changing `Disk usage` to `Spill disk space` makes it more clear?